### ISSUE TITLE
Adjusting target frequency for the whole file

### ISSUE THREAD

**Issue by schtschenok** [HENDRIX-ZT2/pyaudiorestoration#16]
It would be great to have some kind of target frequency adjustment that affects all traces at once, not each trace separately

**Comment by schtschenok** [comment]
Just figured out that this can be done by control+dragging the curve in the upper window. So I guess this one can be closed

**Comment by HENDRIX-ZT2** [comment]
Right that is what I usually do. I thought a 'lock to closest pitch' function could be useful at some point

**Comment by schtschenok** [comment]
By the way, speaking of target frequency, is it possible to have auto-align working for traces that don't intersect at some place on the time axis?

**Comment by HENDRIX-ZT2** [comment]
Yes, indirectly, by setting the same target frequency if they are both selected.

**Comment by schtschenok** [comment]
Ok, I understand, thank you. Maybe you can drop me an email, or reach me on Facebook? I have a few more questions about this software, and I don't really want to create a lot of issues for them here.
tyomamakeev@gmail.com
[Facebook](https://www.facebook.com/tyomamakeev/)
